libs/dyn/convert/struct_info.go

```diff
@@ -84,9 +88,15 @@ func buildStructInfo(typ reflect.Type) structInfo {
 			}
 
 			name, _, _ := strings.Cut(sf.Tag.Get("json"), ",")
-			if name == "" || name == "-" {
+			if name == "" {
 				continue
 			}
+			if name == "-" {
```

Comment:
The `full_name` parameter is a url parameter. Therefore the json is `-`. Added a case that if the `json` is `-` uses the Key name to use get the json name

Reply:
This looks pretty hacky. Can't we use the `url` tag of the field?

Reply:
Instead of this fix that looks at the URL we can come up with something else specifically for monitors.

This fix will break down in other cases so I rather not have a generic solution.